lib/tasks/end_to_end.ex

```diff
@@ -3,6 +3,7 @@ defmodule Mix.Tasks.EndToEnd do
 
   @shortdoc "Runs end-to-end tests"
   def run(_) do
+    chrome_outdated?()
```

Comment:
Do we want to shortcircuit the end-to-end task outright if we determine chromedriver is outdated? It could be an annoying process, if chromedriver is updated frequently (which, I think it is?), but it'd ensure the developer is always grabbing the latest

Reply:
It is certainly something we could do. Does every version of Chrome auto-updating break the tests?

Reply:
Not quite. Any chance it's semantically versioned? Chromdriver and Chrome both follow MAJOR.MINOR.PATCH, and seem to be released in parallel